dev/com.ibm.ws.io.smallrye.graphql/src/io/smallrye/graphql/servlet/IndexInitializer.java

```diff
@@ -35,7 +37,7 @@
  */
 public class IndexInitializer {
 
-    @FFDCIgnore(IOException.class)
+    
```

Comment:
I think we still want the `@FFDCIgnore(IOException.class)` here so that FFDC is not logged here too.  Otherwise, I think these changes look good.

Reply:
I disagree, that is a different IOException to the one which we were originally intending to ignore. That IOException comes from the initial setup of the InputStream. The one we were intending to ignore before was from the `reader.read()` method.

Reply:
Slight correction; the IOException comes from the automatic call to `InputStream.close()` ... if that goes wrong, we want to know about it.